lib/osut/utils.rb

```diff
@@ -553,35 +553,32 @@ def genConstruction(model = nil, specs = {})
   # (May to October in Northern Hemisphere), when outdoor dry bulb temperature
   # is above 18°C and impinging solar radiation is above 100 W/m2.
   #
-  # @param model [OpenStudio::Model::Model] a model
   # @param subs [OpenStudio::Model::SubSurfaceVector] sub surfaces
   #
   # @return [Bool] true if successful
   # @return [Bool] false if invalid input (see logs)
-  def genShade(model = nil, subs = OpenStudio::Model::SubSurfaceVector.new)
+  def genShade(subs = OpenStudio::Model::SubSurfaceVector.new)
```

Comment:
Trimming down OSut method parameter lists, by removing _model_ (when applicable). Only learned recently that every OpenStudio _ModelObject_ has a [model](https://s3.amazonaws.com/openstudio-sdk-documentation/cpp/OpenStudio-3.6.1-doc/model/html/classopenstudio_1_1model_1_1_model_object.html#adc34191db371f238a8035594430d7c5a) method. About a dozen instances ...